shared/reducers/chat2.js

```diff
@@ -429,6 +408,15 @@ const rootReducer = (
 
         s.set('selectedConversation', action.payload.conversationIDKey)
       })
+    case Chat2Gen.unfurlTogglePrompt:
+      return state.updateIn(
+        ['unfurlPromptMap', action.payload.conversationIDKey, action.payload.messageID],
+        prompts => {
```

Comment:
i believe this can be
```
const {show, domain} = action.payload
return state.updateIn(['unfurlPromptMap', action.payload.conversationIDKey, action.payload.messageID], I.Set(), prompts => show ? prompts.add(domain) : prompts.delete(domain)
```

Reply:
Ah yeah, will change.